app/src/main/java/org/mozilla/fenix/perf/ProfilerReusableComposable.kt

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.perf
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+@Composable
+fun DialogueCard(content: @Composable () -> Unit) {
+    Card(
+        elevation = 8.dp,
+        shape = RoundedCornerShape(12.dp)
+    ) {
+        content()
+    }
+}
+
+@Composable
+fun LabeledRadioButton(
+    text: String,
+    subText: String,
+    state: MutableState<String>
+) {
+    Row {
+        RadioButton(
+            selected = state.value == text,
+            onClick = { state.value = text },
```

Comment:
thought (non-blocking): a more robust way to define this would be to have the state represented by an enum. e.g.
```kotlin
fun <T> LabeledRadioButton(
  // ...
  state: MutableState<T>,
  stateWhenSelected: T
) {
  // ...
  RadioButton(
    selected = state.value == stateWhenSelected,
    onClick = { state.value = stateWhenSelected }
  // ...
}

// usage
LabelRadioButton(
  // ...
  threadsAndFeaturesState,
  stateWhenSelected = ProfilerSettings.Firefox
)
```

But we don't need to change this now and can save it for a future iteration of this code.